src/domain/settlement/index.js

```diff
@@ -78,7 +78,7 @@ const groupSettlementWindowContentBySettlementWindow = (records) => {
     if (id in settlementWindows) {
       settlementWindows[id].push(record)
     } else {
-      settlementWindows[id] = record
+      settlementWindows[id] = [record]
```

Comment:
are you sure that "record" should be in an array?

Reply:
100 percent - see line 79